manager/jobs/api/serializers.py

```diff
@@ -105,7 +126,7 @@ class Meta:
             "began",
             "ended",
             "result",
-            "url",
+            "urls",
```

Comment:
Note that this might be an error, the backward compatibility might be broken if `url` is not in response

Reply:
I checked and this is OK, it's a list of `read_only_fields` and `url` since it is derived is by definition read only.